app/services/meursing_measure_component_resolver_service.rb

```diff
@@ -0,0 +1,34 @@
+class MeursingMeasureComponentResolverService
```

Comment:
Is it worth namespacing these? `MeursingMeasure::ComponentServiceResolver`

Reply:
Just to group related services together?

Reply:
Don't get the value of this? To be able to easily remove the meursing measure services?